test/test_group.cpp

```diff
@@ -2508,5 +2508,84 @@ TEST(Group_ArrayCompression_Correctness_Random_Input)
 #endif
 }
 
+TEST(Group_ArrayCompression_Strings)
```

Comment:
I would like to have the same test again, but then committing instead og writing to a new file. Preferably with multiple commits so that we can see that the interner gets updated appropriately